Sources/Runner/Commands/Edit.swift

```diff
@@ -6,19 +6,29 @@ import Logger
 import MarathonCore
 
 func editDanger(logger: Logger) throws {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function body sho...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Edit.swift;
  Line: 8;
-->

- :warning: Function body should span 40 lines or less excluding comments and whitespace: currently spans 42 lines (`function_body_length`)